kebechet/managers/config_initializer/config_initializer.py

```diff
@@ -35,7 +35,17 @@
 _PR_BODY = """## Automatic configuration initialization
 The Kebechet app was installed on this repository but no configuration was
 found.  This PR contains basic configuration of runtime environments and
-managers."""
+managers.
+
+Information about each manager including configuration:
+- [Info](https://thoth-station.ninja/docs/developers/kebechet/managers/info_manager.html)
```

Comment:
maybe also some small description before going into all the links? wdyt?

Reply:
Yeah I can do that